src/dlgTriggerEditor.cpp

```diff
@@ -6966,7 +6966,7 @@ void dlgTriggerEditor::slot_script_main_area_add_handler()
         }
         QListWidgetItem* pItem = mpScriptsMainArea->listWidget_script_registered_event_handlers->currentItem();
         if (!pItem) {
-            return;
+            goto LAZY; //why we have to be this lazy too?
```

Comment:
What does the comment mean?

Reply:
well, it was a reference to above the goto LAZY: that went directly to label LAZY without unrolling out of it. so I do the same here with goto LAZY here to make it work. and.. uh.. I tried to make a joke.. guess not...

Reply:
Instead of doing the lazy way out, could you get rid of the `goto`s? This code is not crazy enough to avoid jumping around...

Reply:
I will try to see if I can redo some of this function.

Reply:
applied.